standalone-metastore/metastore-common/src/test/java/org/apache/hadoop/hive/common/metrics/metrics2/TestCodahaleReportersConf.java

```diff
@@ -149,4 +151,31 @@ public void testNoFallbackOnIncorrectConf() throws Exception {
 
     Assert.assertFalse(jsonReportFile.exists());
   }
+
+  /**
+   * Tests if MetricsFactory is initialised properly, if in case Metrics2Reporter is tried to add more than once.
+   *
+   * **/
```

Comment:
nit: `* **/` looks too much for comment ending

Reply:
okay I will remove extra *